src/theory/quantifiers/instantiate.cpp

```diff
@@ -730,14 +755,16 @@ void Instantiate::getInstantiationTermVectors(
 {
   if (options::incrementalSolving())
   {
-    for (std::pair<const Node, inst::CDInstMatchTrie*>& t : d_c_inst_match_trie)
+    for (const std::pair<const Node, inst::CDInstMatchTrie*>& t :
```

Comment:
I know you really like to be verbose here, but want to give `const auto&` a try here?